sdk/storage/Azure.Storage.Files.DataLake/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 12.20.0-beta.2 (Unreleased)
 
 ### Features Added
+- Added feature parity for DataLake download APIs with Blobs #45418
```

Comment:
"Feature parity for downloads" is a strong blanket statement. Let's maybe cut back and say "Deprecated `Read()`/`ReadAsync()` in favor of `ReadStreaming()`/`ReadStreamingAsync()` and `ReadContent()`/`ReadContentAsync()`.